applications/nrf_desktop/README.rst

```diff
@@ -378,34 +378,39 @@ nRF Desktop build types
-* ``ZDebug`` -- Debug version of the application; the same as the ``ZRelease`` build type, but with debug options enabled.
-* ``ZDebugB0`` -- ``ZDebug`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
-* ``ZDebugMCUBoot`` -- ``ZDebug`` build type with the support for the MCUboot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>` or :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
-* ``ZDebugWithShell`` -- ``ZDebug`` build type with the shell enabled.
+* ``release`` -- Release version of the application with no debugging features.
+* ``release_b0`` -- ``release`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
+* ``release_mcuboot`` -- ``release`` build type with the support for the MCUboot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>` or :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
+* ``debug`` -- Debug version of the application; the same as the ``release`` build type, but with debug options enabled.
+* ``b0`` -- ``debug`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
+* ``mcuboot`` -- ``debug`` build type with the support for the MCUboot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>` or :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
+* ``shell`` -- ``debug`` build type with the shell enabled.
 
 In nRF Desktop, not every development kit can support every build type mentioned above.
 If the given build type is not supported on the selected DK, an error message will appear when `Building and running`_.
-For example, if the ``ZDebugWithShell`` build type is not supported on the selected DK, the following notification appears:
+For example, if the ``shell`` build type is not supported on the selected DK, the following notification appears:
 
 .. code-block:: console
 
-   Configuration file for build type ZDebugWithShell is missing.
+   File not found: ./ncs/nrf/applications/nrf_desktop/configuration/nrf52dmouse_nrf52832/prj_shell.conf
 
 |nrf_desktop_build_type_conf|
-For example, the nRF52840 Development Kit supports the ``ZDebug_keyboard`` configuration, which is defined in the :file:`app_ZDebug_keyboard.conf` file in the :file:`configuration/nrf52840dk_nrf52840` directory.
+For example, the nRF52840 Development Kit supports the ``keyboard`` configuration, which is defined in the :file:`prj_keyboard.conf` file in the :file:`configuration/nrf52840dk_nrf52840` directory.
 This configuration lets you generate the application with the keyboard role.
 
 See :ref:`nrf_desktop_porting_guide` for detailed information about the application configuration and how to create build type files for your hardware.
```

Comment:
Lines 815-832 need to be updated accordingly with the new build type usage steps, also including the VSC extension.

Reply:
In other words, we need to update gs_modifying.rst file.

Reply:
Discussed offline. These documentation changes will be done as a separate PR.